118.md

```diff
@@ -0,0 +1,89 @@
+# NIP-118
+
+## Polls
+
+`draft` `optional`
+
+This NIP defines the event scheme that describe Polls on nostr.
+
+The NIP makes the following tradeoffs.
+
+1. All votes are public.
+2. All responses are editable (latest by timestamp considered last response)
+
+## Events
+
+### Poll Content
+
+The poll event is defined as a `kind:1068` event. Major tags in the poll event are:
+
+- **label**: The accompanying text of the poll.
```

Comment:
Why not use the `content`?

Reply:
it makes it easier if we want to take the same polls format and encrypt it as a private poll, There are no private polls yet however, so i'm open to move this to a label

Reply:
I prefer the `content` unless the specification for private polls is clear.